frontend/src/utils/api.ts

```diff
@@ -9,5 +9,6 @@ export const api = {
     EXPORT: '/api/export/',
     CHANGE_PASSWORD: '/api/change_password/',
     VIEW_SCHEDULED_EVENTS: '/api/scheduled_event/',
+    FETCH_RUN_INSTANCES: '/api/scheduled_event/id/run_instances/',
```

Comment:
This is risky ... template strings must take a risk free format ... e.g `{ID}` or `%ID%`, something you won't usually find in a random string. Now simply having it as `id` ... imagine the url was `/api/cupid/id/smells/stupid` ... doing a string.replace on that aiming to replace `id` with `MEH` would lead to `/api/cupMEH/MEH/smells/stupMEH`